BlocNotes/BlocNotes/DetailNoteViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  DetailNoteViewController.m
+//  BlocNotes
+//
+//  Created by Eddy Chan on 6/26/16.
+//  Copyright © 2016 chan-e. All rights reserved.
+//
+
+#import "DetailNoteViewController.h"
+#import "CoreDataStack.h"
+#import "Note.h"
+
+@interface DetailNoteViewController () <UITextViewDelegate>
+
+@property (weak, nonatomic) IBOutlet UILabel *placeholderTextLabel;
+@property (weak, nonatomic) IBOutlet UITextView *bodyTextView;
+
+@end
+
+@implementation DetailNoteViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    self.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc]
```

Comment:
Ah, you were trying to remove the title. I would just leave it as is.

> On Jul 1, 2016, at 9:25 PM, Eddy Chan <notifications@github.com> wrote:
> 
> In BlocNotes/BlocNotes/DetailNoteViewController.m:
> 
> > +#import "Note.h"
> > +
> > +@interface DetailNoteViewController () <UITextViewDelegate>
> > +
> > +@property (weak, nonatomic) IBOutlet UILabel *placeholderTextLabel;
> > +@property (weak, nonatomic) IBOutlet UITextView *bodyTextView;
> > +
> > +@end
> > +
> > +@implementation DetailNoteViewController
> > +
> > +- (void)viewDidLoad {
> > +    [super viewDidLoad];
> > +    // Do any additional setup after loading the view.
> > +    
> > +    self.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc]
> I saw the wireframes for this user story, and the back button had no title. Do you want me to keep the default back button title i.e. "< BlocNotes" ?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>